site/docs/uk/plugins/autoquote.md

```diff
@@ -6,10 +6,13 @@ next: false
 # Автоматичне встановлення відповіді
 
 Іноді необхідно завжди надсилати повідомлення як відповіді, особливо для ботів, призначених для використання в групах.
-Зазвичай ми робимо це, додаючи параметр `reply_to_message_id` до методів, які надсилають повідомлення: `sendText`, `reply`, `sendPhoto`, `replyWithPhoto` тощо.
+Зазвичай ми робимо це, додаючи параметр `reply_parameters` до методів, які надсилають повідомлення: `sendText`, `reply`, `sendPhoto`, `replyWithPhoto` тощо.
 Однак, якщо ви робите це для кожного повідомлення окремо, це може бути трудомістким і нудним.
 
-Цей плагін встановлює параметр `reply_to_message_id` зі значенням `ctx.msg.message_id` для всіх методів `reply*` та `send*`, які підтримують його, щоб кожне повідомлення ставало відповіддю на повідомлення, яке його викликало.
+Цей плагін встановлює застарілий параметр `reply_to_message_id` зі значенням `ctx.msg.message_id` для всіх методів `reply*` та `send*`, які підтримують його, щоб кожне повідомлення ставало відповіддю на повідомлення, яке його викликало.
+
+> Параметр `reply_to_message_id` все ще підтримується Bot API для зворотної сумісності.
+> У новому коді слід використовувати `reply_paramters`.
```

Comment:
```suggestion
> У новому коді слід використовувати `reply_parameters`.
```

Reply:
This should be corrected in all languages after the merge of this PR. @KnorpelSenf

Reply:
Copy. @niusia-ua